cachito/web/purl.py

```diff
@@ -101,6 +100,33 @@ def _to_purl_pip(package):
         return f"pkg:generic/{name}?download_url={quoted_url}&checksum={checksum}"
 
 
+def _to_purl_rubygems(package):
+    if package.version and package.version.startswith("./"):
+        # Package is relative to the parent module
+        normpath = os.path.normpath(package.version.removeprefix("./"))
+        return f"{PARENT_PURL_PLACEHOLDER}#{normpath}"
+    parsed_url = urllib.parse.urlparse(package.version)
+    if parsed_url.scheme.startswith("git+"):
+        # refactor it out from pip
```

Comment:
IMO this comment is not really necessary, it won't make sense out of the context of this PR

Reply:
Yes, sorry, this was a TODO comment for myself